standalone/src/main/java/io/pravega/local/LocalPravegaEmulator.java

```diff
@@ -125,12 +126,12 @@ public void run() {
 
             log.info("Starting Pravega Emulator with ports: ZK port {}, controllerPort {}, SegmentStorePort {}",
                     conf.getZkPort(), conf.getControllerPort(), conf.getSegmentStorePort());
-
             localPravega.start();
-
-            System.out.println(
-                    String.format("Pravega Sandbox is running locally now. You could access it at %s:%d", "127.0.0.1",
-                            conf.getControllerPort()));
+            String msg = String.format("# Pravega Sandbox is running locally now. You could access it at %s:%d. #", "127.0.0.1", conf.getControllerPort());
```

Comment:
A change that is less needed with the changes performed in `logback.xml` but this further brings attention to pravega-standalone is in a ready state.